static/canjs.js

```diff
@@ -162,7 +162,9 @@ function init() {
 			// Move the social-side-container to inside the new sidebar
 			var sidebarElement = document.querySelector('canjs-sidebar');
 			var socialContainer = currentMenu.querySelector('.social-side-container');
-			sidebarElement.insertBefore(socialContainer, sidebarElement.firstChild);
+			var bitoviContainer = currentMenu.querySelector('.by-bitovi-container');
+			sidebarElement.insertBefore(socialContainer, sidebarElement.secondChild);
+			sidebarElement.insertBefore(bitoviContainer, sidebarElement.thirdChild);
```

Comment:
This code:

```js
sidebarElement.insertBefore(socialContainer, sidebarElement.firstChild);
```

…will take the `.social-side-container` and insert it before the sidebar element’s `<ul>`, so those links appear above the main nav.

Now, I think it should be something like:

```js
sidebarElement.appendChild(socialContainer);
sidebarElement.appendChild(bitoviContainer);
```

That should add those two containers as children to the sidebar element, after the `<ul>`.

[There’s no such thing as `secondChild` or `thirdChild`.]

Reply:
This is what happens when I get curious about how JS works :P